amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpConnector.scala

```diff
@@ -15,19 +15,42 @@ private[amqp] trait AmqpConnector {
     val factory = new ConnectionFactory
     settings match {
       case AmqpConnectionUri(uri) => factory.setUri(uri)
-      case AmqpConnectionDetails(host, port, maybeCredentials, maybeVirtualHost, sslProtocol) =>
-        factory.setHost(host)
-        factory.setPort(port)
+      case AmqpConnectionDetails(hostAndport, maybeCredentials, maybeVirtualHost, sslProtocol, requestedHeartbeat,
+          connectionTimeout, handshakeTimeout, shutdownTimeout, networkRecoveryInterval, automaticRecoveryEnabled,
+          topologyRecoveryEnabled, exceptionHandler) =>
+        if (hostAndport.size == 1) {
+          factory.setHost(hostAndport.head._1)
+          factory.setPort(hostAndport.head._2)
+        }
```

Comment:
If more than one host and port provided, then no host nor port will be set on factory?